### PR TITLE
Fixes prisoner icon on orbit ui

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
GBP No update please.
Going for a hotfix speedrun here. 40 minutes since merge.

From
Screenshot 2022-10-19 195505
To
Screenshot 2022-10-19 200243

## Why It's Good For The Game
Prisoners get an icon.

## Changelog

:cl:
fix: Fixed prisoner icon on orbit menu.
/:cl:

### CODE REVIEW

**Review by ShizCalev — APPROVED** [review]

File: code/datums/id_trim/jobs.dm [hunk 1]
```diff
@@ -722,7 +722,7 @@
 /datum/id_trim/job/prisoner
 	assignment = "Prisoner"
 	trim_state = "trim_warden"
-	orbit_icon = "lock-keyhole"
+	orbit_icon = "lock"
```
Comment (MrDoomBringer) [inline code/datums/id_trim/jobs.dm:725]:
```suggestion
	orbit_icon = "container-storage"
```
image
makes a little more sense imo

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/datums/id_trim/jobs.dm (hunk 1 above)
Comment (jlsnow301) [reply code/datums/id_trim/jobs.dm:725]: Agree! Just need to test it, not all icons work even though they should (same issue here!)

Comment (Tastyfish) [reply code/datums/id_trim/jobs.dm:725]: Won't work, the whole problem is we have the free version of FontAwesome, you need to press the Free filter when looking through them.

Comment (MrDoomBringer) [reply code/datums/id_trim/jobs.dm:725]: Good catch haha, thought the issue was a different version of FA

**Pull request merged**